examples/low-level-api/CMakeLists.txt

```diff
@@ -15,7 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
+include_directories(SYSTEM . )
```

Comment:
Please use https://cmake.org/cmake/help/v3.3/command/target_include_directories.html so that this does not spread to the main library.

Reply:
+1. See example here: https://github.com/apache/arrow/blob/master/cpp/src/arrow/python/CMakeLists.txt#L121